Makefile

```diff
@@ -113,9 +108,8 @@ DEFAULT_IMAGES += $(MLRUN_LEGACY_MODELS_IMAGE_NAME)
 models-legacy: ## Build models legacy docker image
 	docker build \
 		--file dockerfiles/models/$(MLRUN_LEGACY_DOCKERFILE_DIR_NAME)/Dockerfile \
-		--build-arg MLRUN_PACKAGE_TAG=$(MLRUN_PACKAGE_TAG) \
 		--build-arg MLRUN_MLUTILS_GITHUB_TAG=$(MLRUN_MLUTILS_GITHUB_TAG) \
-		--build-arg MLRUN_GITHUB_REPO=$(MLRUN_GITHUB_REPO) \
+		--build-arg MLRUN_MLUTILS_CACHE_DATE="$$(date)" \
```

Comment:
hmm, assuming  I dont need this _hack_ I would be happy to know I can unhack this deoptimization.
In human words, please use a VAR `MLRUN_MLUTILS_CACHE_DATE ?= $$(date)` that will allow me to
`MLRUN_MLUTILS_CACHE_DATE=initial make build` and get my docker cache working as usual

Reply:
Great idea, done